tests/testthat/test-generate_pt_nowcast_mat_list.R

```diff
@@ -90,12 +87,19 @@ test_that("Default n_history_delay uses minimum rows", {
   expect_false(all(result_default[[1]] == result_custom[[1]]))
 })
 
-### Test 3: Custom n_history_delay
 test_that("Custom n_history_delay is respected", {
-  expect_no_error(generate_pt_nowcast_mat_list(
+  result <- generate_pt_nowcast_mat_list(
     reporting_triangle_list = retro_rts_list,
     n = 5
-  ))
+  )
+  # Compare to estimate using n=5
+  first_triangle <- result[[1]]
+  delay_pmf <- get_delay_estimate(retro_rts_list[[1]],
+    n = 5
+  )
+  exp_first_triangle <- apply_delay(retro_rts_list[[1]], delay_pmf)
+  expect_equal(first_triangle, exp_first_triangle, tol = 0.0001)
+
   expect_error(generate_pt_nowcast_mat_list(
```

Comment:
why just change one of these? seems safe to remove. Minor so happy to merge as is.

Reply:
Sorry I don't think I am following -- this adds a test that makes sure the first triangle is estimated from an`n_history_delay = 5` by comparing to an isolated instance of estimating a delay and applying it to just the first triangle (just making sure it works correctly within the nested function of `generate_pt_nowcast_mat_list()`

Reply:
you have other changes that then aren't tested but just checked they don't error. If the default was 5 for some reason this would introduce a test gap.